src/app/components/pokemon-list/pokemon-list-item/pokemon-list-item.component.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../../../styles.scss';
```

Comment:
try to remove style.scss from all the scss files

Reply:
its must